site/public/js/ta-grading-v2.js

```diff
@@ -330,6 +330,29 @@ function updateCookies(){
 
 //-----------------------------------------------------------------------------
 // Student navigation
+function gotoMainPage() {
+
+  let selector;
+  let window_location;
+
+  selector = "#main-page";
+  window_location = $(selector)[0].dataset.href
+
+  console.log(window_location);
```

Comment:
I think you can make this a little compact by removing the `selector` variable (not used anywhere other $(selector)[0].dataset.href), then declaring the `window_location` at the same line and removing the console line (used for testing). Thanks :)